app/main.py

```diff
@@ -1,3 +1,87 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import datetime
+import math
+from app.car import Car
+from app.shop import Shop
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as config:
+        user_data = json.load(config)
+
+    customers_data = user_data["customers"]
+    shops = user_data["shops"]
+    price_of_fuel = user_data["FUEL_PRICE"]
+
+    customers = []
+    for customer_data in customers_data:
+        car = Car(
+            brand=customer_data["car"]["brand"],
+            fuel_consumption=customer_data["car"]["fuel_consumption"],
+        )
+
+        customer = Customer(
+            name=customer_data["name"],
+            product_cart=customer_data["product_cart"],
+            money=customer_data["money"],
+            car=car,
+            customer_location=customer_data["location"],
+        )
+        customers.append(customer)
+
+    shops_list = [Shop(**shop_data) for shop_data in shops]
```

Comment:
```suggestion
    shops = [Shop(**shop_data) for shop_data in shops]
```
This suffix is redundant